core/src/main/scala/com/databricks/labs/remorph/Dag.scala

```diff
@@ -0,0 +1,30 @@
+package com.databricks.labs.remorph
+import scala.collection.mutable
+
+case class Node(name: String)
+
+case class Edge(from: Node, to: Node)
+
+class DAG {
```

Comment:
DAG is too generic name not to have generic parameters.

Reply:
how about `TableLineage` ?